### ISSUE TITLE
Questions about the labeled`UniDA` with development/test sets. 

### ISSUE THREAD

**Issue by jianguoz** [siat-nlp/GALAXY#8]
In Table 1 and Readme, I found that among the 975780 utterances, some datasets such as MultiWOZ and SimJoint also use development set and test set during pre-training. 

But the paper further evaluates on MultiWOZ test set. Would it make the evaluations unfair as the model already uses partial labeled information of MultiWOZ?

**Comment by HwwAncient** [comment]
Hello, among the 975780 utterances, UniDA only contains the training data of MultiWOZ but all data (train/dev/test) of the rest seven datasets. So it is fair to make the evaluations on the downstream tasks on MultiWOZ/CamRest/In-Car Assistant datasets.

BTW, we offer the original dialog ID for every dialog sample in the UniDA. You can also check it according to the offered ID.
Thanks!

**Comment by jianguoz** [comment]
Thanks very much for your clarification and valuable information:)
